### DISCUSSION TITLE
364p 실습 6-11 질문드립니다.

### DISCUSSION THREAD

**Discussion by dkyou7** [fsoftwareengineer/todo-application#121] [질문과 답변]
![image](https://user-images.githubusercontent.com/26649731/171986809-5eddda48-1453-4c13-8bc6-e6bb078baba6.png)

- eb create --database --elb-type application --instance-type t2.micro 시, 해당 오류가 나오는데 원인을 잘 모르겠어서 질문드립니다.

**Comment by wjdtj9656** [comment]
패스워드 입력안하신듯
